app/views/energy_tariffs/energy_tariff_charges/_charge_vat.html.erb

```diff
@@ -10,7 +10,7 @@
       <%= t('schools.user_tariff_charges.charge_vat.rate') %>
     </td>
     <td class="value">
-      <%= f.select :vat_rate, options_for_select(['0%', '5%', '20%'], energy_tariff.vat_rate), {required: :true}, { class: 'form-control' } %>
+      <%= f.select :vat_rate, options_for_select(['0', '5', '20'], energy_tariff.vat_rate), {required: :true}, { class: 'form-control' } %>
```

Comment:
Is this an intentional change here? There's a ticket to review the VAT select to allow an empty option, but I think the labels should still be %?